<launch>
    <node pkg="autonomy_leg_detection" type="leg_detection" name="leg_detection" output="screen">
        <param name="show_viz" value="false"/>
        <param name="show_marker" value="true"/>
        <param name="laserFeature/arc_min_aperture" value="1.57"/>
        <param name="laserFeature/arc_max_aperture" value="2.375"/>
        <param name="laserFeature/arc_std_max" value="0.05"/>
        <param name="laserFeature/segmentation_threshold" value="300.0"/>
        <param name="laserFeature/line_min_distance" value="170.0"/>
        <param name="laserFeature/line_error_threshold" value="5.0"/>
        <param name="laserFeature/max_leg_diameter" value="200.0"/>
        <param name="laserFeature/min_leg_diameter" value="50.0"/>
        <param name="featureLegTracker/leg_clean_ticks" value="2.0"/>
        <param name="featureLegTracker/person_clean_ticks" value="2.0"/>
        <param name="featureLegTracker/leg_update_radius" value="500.0"/>
        <param name="featureLegTracker/person_radius" value="500.0"/>
        <param name="marker_scale" value="0.5"/>
        <remap from="/scan" to="/scan_filtered" />

    </node>

    <node pkg="laser_filters" type="scan_to_scan_filter_chain" output="screen"      name="laser_filter">
        <remap from="scan" to="lidar/front/scan" />
    </node>
</launch>
